notes/src/Components/deletenote.js

```diff
@@ -0,0 +1,25 @@
+import  React, {Component} from 'react';
+import {Route} from 'react-router-dom';
+import Note from './note'
+//import axios from 'axios';
+
+export default class DeleteNote extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            movie: []
```

Comment:
movie???? I believe this component was for testing purposes and could be safely removed.